source/frontend/src/features/properties/map/acquisition/common/update/acquisitionTeam/UpdateAcquisitionTeamYupSchema.ts

```diff
@@ -1,6 +1,13 @@
 import * as Yup from 'yup';
 
 export const UpdateAcquisitionTeamYupSchema = Yup.object().shape({
+  fundingTypeOtherDescription: Yup.string().when('fundingTypeCode', {
+    is: (fundingTypeCode: string) => fundingTypeCode && fundingTypeCode === 'OTHER',
+    then: Yup.string()
+      .required('Other Description required')
```

Comment:
should this be Other Description or Other description.

Or perhaps it should refer to the field name which is: **Other funding**

Reply:
could be. Just followed what we do now